research-hub-web/src/app/components/subhubs/subhubs.component.spec.ts

```diff
@@ -143,7 +143,7 @@ describe('SubhubsComponent', () => {
       }
     ],
     "__typename": "SubHubCollection"
-  } as SubHubCollection);
+  } as unknown as SubHubCollection);
```

Comment:
I also had to add `as unknown` to bypass some overlap issues but I still don't know why